plugins/techdocs/src/reader/components/TechDocsNotFound.test.tsx

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { TechDocsNotFound } from './TechDocsNotFound';
+import React from 'react';
+import { render } from '@testing-library/react';
+import { wrapInTestApp } from '@backstage/test-utils';
+
+describe('TechDocs Not Found', () => {
+  it('should render a Documentation not found page', async () => {
+    const { queryByText } = render(wrapInTestApp(<TechDocsNotFound />));
+    expect(queryByText(/Error: Documentation not found/i)).toBeInTheDocument();
```

Comment:
`Error` and `Documentation` can be lowercase as you are using `i` in your `/{regex}/i` which is for insensitive casing.

```suggestion
    expect(queryByText(/error: documentation not found/i)).toBeInTheDocument();
```